input_files/CHP/chp_input_defaults.py

```diff
@@ -0,0 +1,154 @@
+import pandas as pd
+import numpy as np
+import os
+import copy
+import pickle
+
+# Lower and upper bounds for size classes - Class 0 is the total average across entire range of data
+class_bounds = {"recip_engine": [(30, 10000), (30, 100), (100, 630), (630, 3300), (3300, 10000)],
+                "micro_turbine": [(30, 1000), (30, 60), (60, 190), (190, 1000)],
+                "combustion_turbine": [(950, 20000), (950, 3300), (3300, 20000)],
+                "fuel_cell": [(30, 10000), (30, 320), (320, 1400), (1400, 10000)]}
+
+# Count number of size classes for each prime mover
+n_classes = {pm: len(class_bounds[pm]) for pm in class_bounds.keys()}
+
+# If no size class is input, use these defaults for the API (see validators.py).
+default_chp_size_class = {"recip_engine": 0,
+                          "micro_turbine": 0,
+                          "combustion_turbine": 0,
+                          "fuel_cell": 0}
+
+# Fraction of lower bound kW for size class that is assigned to min_allowable_kw
+min_allow_frac = [0.5, 0.7, 1.0, 0.5]
+
+# Assumptions used to get the half load performance from the full load
+elec_effic_half_frac = [0.9, 0.8, 0.8, 0.9]
+hre_half_frac = [1.0, 1.0, 1.0, 1.0]
+
+chp_defaults_not_size_class_dependent = {'recip_engine': {
```

Comment:
@NLaws This is where the CHP default inputs data currently lives. This is a combination of raw input values, values used to calculate the input values from the .csv data (also in this input_files.CHP folder), and a script to process the .csv data and create a .pickle file which stores all the processed data which is ready for reading directly. We then read in the .pickle file in validators.py to assign defaults and override defaults with user-entered values. We anticipate that the CHP default data will change in the next few months, so I wanted the raw data and processing calcs in one place (and not clutter other files because it's a lot).